www/js/rover-api.js

```diff
@@ -0,0 +1,98 @@
+/*----- MISC GLOBALS -----*/
+var roverDomain = ''; //Later, if the rover is not hosting the webpage, its address will go here
+var roverApiPath = '/api/v1/';
+
+/*----- HELPER FUNCTIONS -----*/
+function roverResource(resource) {
+	return roverDomain+roverApiPath+resource;

```

Comment:
I think this could be a nice utility function if you extended it to accept arrays, something like
` return roverDomain + roverApiPath + (Array.isArray(resource) ? resource.join('/') : resource);`
Then line 47 could be:

`$.get(roverResource(['blockdiagrams', name]), function(response) {...}`